CMakeLists.txt

```diff
@@ -285,14 +268,7 @@ target_compile_options(compiler_flags INTERFACE -DARCH_${ARCH_NAME_DEF})
 
 add_library(metal_header_directories INTERFACE)
 target_include_directories(metal_header_directories INTERFACE ${PROJECT_SOURCE_DIR}/tt_metal/hw/inc)
-target_include_directories(
-    metal_header_directories
-    SYSTEM
-    INTERFACE
-        ${reflect_SOURCE_DIR}
-        ${flatbuffers_include_dir}
-        ${nng_include_dir}
```

Comment:
Great to see these two deleted.  Were they already dead?  I don't see them added as deps elsewhere in this PR.

Reply:
These were the two vars that were backpropagated from UMD repo into parent scope.

New UMD repo does not try to do that.

Include paths can propagate with the target, if necessary.

Reply:
Ouch.  So this code existed in tt-metal purely for umd 🙃 
Thanks!